docs/cli/open.md

```diff
@@ -0,0 +1,19 @@
+open
+====
+
+Opens a started app in your default browser. Truly for the lazy among us :)
+
+> #### Hint::Containers are cached!
+>
+> If you start an app with a new service or container it will need to pull that container image down. This can take a moment depending on your internet connection. Subsequent pulls to that container or service are cached so they should be much faster.
+
+Usage
+-----
+
+```bash
+# Opens a reasonable guess at your default url
+lando open
+
+# Opens a specific URL
+lando open --url https://myapp.lndo.site
```

Comment:
Is the `--url` switch necessary?  I'm even lazier than you think. ;)

Reply:
Right, but do you even need it with the override?  Can't you just use the domain as the argument?

Reply:
Ahh; I see; you are probably right; I'll check

Reply:
@serundeputy dropping a note here: might want to test this against non-recipes to get a sense of the issues we are up against